cmd/source/controller/main.go

```diff
@@ -196,5 +198,8 @@ func main() {
 		binding.NewController, NewKafkaBindingWebhook(kfkSelector),
 
 		source.NewController,
+
+		// Sources CRD
+		sourcecrd.NewController,
```

Comment:
@lionelvillard I fear/guess something like this must be done for all sources?

Reply:
this shouldn't be need. This controller is already included in the eventing controller

Reply:
I'm getting this in the eventing controller log, when setting the proper duck type label on the KafkaSource CRD:

```
{"level":"error","ts":"2021-12-07T15:38:50.632Z","logger":"controller","caller":"duck/duck.go:78","msg":"runtime object is not convertible to Source duck type: ","commit":"d5201e7","knative.dev/pod":"eventing-controller-c6c8b75dd-zh9xs","knative.dev/traceid":"fa0b199c-9bff-42bf-a99d-77683789b0a6","knative.dev/key":"sources-kafka-sanity/fruits","runtimeObj":null,"stacktrace":"knative.dev/eventing/pkg/reconciler/source/duck.(*Reconciler).Reconcile\n\tknative.dev/eventing/pkg/reconciler/source/duck/duck.go:78\nknative.dev/pkg/controller.(*Impl).processNextWorkItem\n\tknative.dev/pkg@v0.0.0-20211111114938-0b0c3390a475/controller/controller.go:542\nknative.dev/pkg/controller.(*Impl).RunContext.func3\n\tknative.dev/pkg@v0.0.0-20211111114938-0b0c3390a475/controller/controller.go:478"}
{"level":"info","ts":"2021-12-07T15:38:50.632Z","logger":"controller","caller":"controller/controller.go:550","msg":"Reconcile succeeded","commit":"d5201e7","knative.dev/pod":"eventing-controller-c6c8b75dd-zh9xs","knative.dev/traceid":"fa0b199c-9bff-42bf-a99d-77683789b0a6","knative.dev/key":"sources-kafka-sanity/fruits","duration":0.000156068}
```

Reply:
I'm guessing this is a RBAC issue.

Reply:
/hold
let's not merge... but not sure on actual detail, atm